scene/3d/ray_cast.h

```diff
@@ -49,6 +50,11 @@ class RayCast : public Spatial {
 	uint32_t layer_mask;
 	uint32_t type_mask;
 
+	Node *debug_shape;
+	Ref<FixedSpatialMaterial> line_material;
```

Comment:
Shouldn't be a member variable. Just put it here temporarily to reduce the amount of leaking so it could be tested.